### PR TITLE
web: added z-index on the suggestions

### LINKED ISSUE #7611: Widening file tree sidebar on top of search suggestions
- **Sourcegraph version:** Sourcegraph.com

image

#### Steps to reproduce:

1. Go to a file
2. Type in the search bar to get autocomplete suggestions
3. See that you can widen the sidebar on top of the suggestion

### PR DESCRIPTION
**Pull request by Akarshit**
Closes #7611 

I have added a z-index to the suggestion dropdown so that it stays on top of the windows below.

### CODE REVIEW

**Comment by sqs** [comment]
Thank you for the contribution! :tada:  Someone from the @sourcegraph/web team will review it soon. It looks like it does work, but they might have some other tips.

File: web/src/search/input/QueryInput.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .query-input2 {
     flex: 1 1 auto;
     position: relative;
+    z-index: 2 !important;
```
Comment (lguychard) [inline web/src/search/input/QueryInput.scss]: Is the `!important` necessary here?

**Comment by felixfbecker** [comment]
This needs to be tested against other overlays, e.g. the hover overlay

**Comment by felixfbecker** [comment]
Closing since this search input doesn't exist anymore and this issue does not reproduce on the smart search input.

**Pull request closed without merging**